exploration/open-close-expressions.md

```diff
@@ -220,3 +220,68 @@ Rendered as React this could become:
 _What other solutions are available?_
 _How do they compare against the requirements?_
 _What other properties they have?_
+
+### Do Nothing
+
+We could not provide any special support for spans and markup elements,
+and instead delegate the problem completely to tools and downstream processing layers.
+
+```
+This is <html:strong>bold</html:strong> and this is <html:img alt="an image">.
+```
+
+#### Pros:
+
+* No work required from us right now.
+* It's HTML.
+* The least surprising syntax for developers and translators.
+* Some CAT tools already support HTML in translations.
+
+#### Cons:
+
+* Requires quoting in XML-based containers.
```

Comment:
But this isn't any different from a string that happens to contain any of the syntax characters.

Reply:
Yes, but I think this is worth calling out explicitly because:

* HTML markup in translations is common, and
* storing translations in XML-based containers is common.

The combination of these two points means that if the syntax to get HTML markup in MF2 is the HTML syntax (because there's no alternative), then you'll get a lot of `&lt;` in XML-based containers.